Resources/Prototypes/Voice/speech_emotes.yml

```diff
@@ -127,6 +127,9 @@
   id: Weh
   category: Vocal
   chatMessages: [ wehs ]
+  chatTriggers:
```

Comment:
add #Floof tag so it's not missed in upstream merges

Reply:
oh, oops, i was coding it in really quick